### PR TITLE
useContext experiment

### PR DESCRIPTION
**Pull request by BrettEastman**
Please review and add comments.

### CODE REVIEW

File: client/source/App.jsx [hunk 1]
```diff
@@ -28,16 +29,19 @@ const App = () => {
...
   return (
     <div>
-      <Header />
-      <Overview />
-      <button onClick={testGetRequest}>TEST GET PRODUCTS REQUEST</button>
-      <RelatedList />
-      <OutfitList />
-      <QAModule />
-      <ReviewModule />
+      <ProdContext.Provider value={{prodID, setProdID}}>
```
Comment (avinashi10) [inline client/source/App.jsx:37]: Have you thought about moving the `.Provider` method to the index.jsx file so it only needs to wrap around the <App /> component and there will be less merge conflicts as we rearrange the `app.jsx` file

File: client/source/App.jsx [hunk 2]
```diff
@@ -28,16 +29,19 @@ const App = () => {
...
+        <Header />
+        <Overview />
+        <button onClick={testGetRequest}>TEST GET PRODUCTS REQUEST</button>
+        <RelatedList />
+        <OutfitList />
+        <QAModule />
+        <ReviewModule />
+      </ProdContext.Provider>
```
Comment (avinashi10) [inline client/source/App.jsx:45]: see line 36 comment

File: client/source/App.jsx [hunk 3]
```diff
@@ -28,16 +29,19 @@ const App = () => {
       });
   }
 
+  const [prodID, setProdID] = useState('product id');
```
Comment (avinashi10) [inline client/source/App.jsx:33]: if you use the suggestion I made below regarding the `.Provider` method, you'll need to `useState` / `setState` in the `index.jsx` file as well

**Review by avinashi10 — COMMENTED** [review]
One small syntax suggestion that August taught me - every file should end in a blank line (not a line of code) as this can prevent conflicts with older systems which used to require this!

File: client/source/ProdContext.js [hunk 4]
```diff
@@ -0,0 +1,3 @@
+import { createContext } from 'react';
+
+export const ProdContext = createContext(null);
```
Comment (AedonGrunGott) [inline client/source/ProdContext.js:3]: line endings, yo! We'll fix this later though.

**Review by AedonGrunGott — APPROVED** [review]

**Pull request merged**